internal/examples/config/client.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/kr/pretty"
+
+	"go.uber.org/yarpc/encoding/json"
+)
+
+func main() {
+	config, err := os.Open("client.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer config.Close()
+
+	builder, err := loader.LoadYAML(config)
+	if err != nil {
+		log.Fatal(err)
+	}
+	pretty.Println("Loaded configuration:", builder)
+
+	d, err := builder.BuildDispatcher()
```

Comment:
how do I insert a custom outbound?  Should we expose the config directly here?  Do we need to go through a builder object or can we jump straight to a yarpc.Config in loader.LoadYAML?